code/modules/food_and_drinks/recipes/drinks/drinks_non-alcoholic.dm

```diff
@@ -170,3 +170,7 @@
 	results = list(/datum/reagent/consumable/cucumberlemonade = 5)
 	required_reagents = list(/datum/reagent/consumable/lemon_lime = 3, /datum/reagent/consumable/cucumberjuice = 2, /datum/reagent/consumable/ice = 1)
 
+/datum/chemical_reaction/drink/mississippi_queen
+	results = list(/datum/reagent/consumable/mississippi_queen = 50)
+	required_reagents = list(/datum/reagent/consumable/tomatojuice = 15, /datum/reagent/consumable/mayonnaise = 10, /datum/reagent/consumable/soysauce = 5, /datum/reagent/consumable/vinegar = 2, /datum/reagent/consumable/capsaicin = 10, /datum/reagent/consumable/coco = 2)
```

Comment:
Not a big fan of this recipe.  Most drinks are 2-3 reagents (in rare cases 5) and the results are usually 3, 5, or 10 units.

I'd recommend using a mixed drink as one reagent ie. `hot_coco` and two other regular reagents.  Lower the result amount to be similar to other drinks.

Reply:
Shit, I gotta think on what to use for some of them then. I can't think of anything that would have similar reagents, and I remember I was getting yelled at for actually leaving some reagents _out_ before

Reply:
I brought it up with armhulenn and they said reducing it further would be pushing the reference further away from what it's referencing. I don't really know who's advice to take here lol